### PR TITLE
testing against ruby 2.2.0

### PR DESCRIPTION
**Pull request by lucianosousa**

### CODE REVIEW

**Comment by justinfrench** [comment]
@lucianosousa are you looking into the failures here?

File: .travis.yml [hunk 1]
```diff
@@ -26,3 +27,7 @@ matrix:
     - rvm: 2.1
       gemfile: gemfiles/rails_edge.gemfile
       env: DEFER_GC=false RAILS_EDGE=true
+    - rvm: 2.2.0
+      gemfile: gemfiles/rails_3.2.gemfile
```
Comment (oliverklee) [inline .travis.yml]: Does Rails 3.2 work with Ruby 2.2?

Comment (lucianosousa) [inline .travis.yml]: still not. look at: https://github.com/rails/rails/pull/18306

Comment (oliverklee) [inline .travis.yml]: Then we should have this as an allowed failure.

For the page_title_helper, we did it like this:
https://github.com/lwe/page_title_helper/blob/master/.travis.yml

**Comment by pranas** [comment]
This is no longer relevant after #1170 was merged in. /cc @mikz

**Comment by mikz** [comment]
This was fixed by #1170

**Pull request closed without merging**